CHANGELOG.rst

```diff
@@ -4,6 +4,9 @@
 
 **Breaking changes**:
 
+- The ``TIME_UNSPECIFIED`` value, which is used to indicate that frequency should be
```

Comment:
Did we actually release a version of tsinfer with this constant in the public API? If so, we need to be a bit more careful. If not, then there's no point in putting it in the changelog.

Reply:
Yeah, good point. It wasn't in 0.1.4, which it the last tagged release, so I guess I can just delete that from the changelog?

Reply:
Yeah, that makes things a lot simpler. We can delete this from the changelog.

But, we should probably bump the file version number so that files we have lying around at the moment don't get misinterpreted.

Reply:
Yep. I was going to bump the number once we had added flags into the individuals, and (possibly) switched to `null` in the metadata fields.

Reply:
Removed from changelog. Version bump later today, I guess?